### ISSUE TITLE
[Perfective] PRI error message is not sufficient

### ISSUE THREAD

**Issue by AminaSalah** [CDH-Studio/UpSkill#1519]
### [Perfective] PRI error message can be revised
The PRI error message can be revised. It only says “Must be a valid PRI” but does not let the user know what is the required format. The telephone numbers on the same page do provide the user with the required format. PS the field which has the error are highlighted red and the error message is also red which is good

**Issue Location:** 
- Create Profile: Contact Information Form
- Edit Profile: Contact Information Form

**Steps to recreate:**
- Navigate to the Create Profile: Contact Information Form or  Edit Profile: Contact Information Form
- In the PRI text field type anything that is not an 8 digit number
- The error message will appear

EXP: The error message is descriptive and helps the user determine the expected input for the field
ACTUAL: The error message is missing the required format.

**Comment by AminaSalah** [comment]
Btw I am not able to add labels to the issue @dynamic11. If you have any questions about this issue as well let me know

**Comment by MenaMachado** [comment]
Recommend we add an eg. in the error message as a solution
